Lib9c/GameConfig.cs

```diff
@@ -40,6 +40,8 @@ public static class GameConfig
         public const int ArenaScoreDefault = 1000;
         public const int ArenaChallengeCountMax = 5;
 
+        public const int HardStageBeginId = 101;
```

Comment:
https://github.com/planetarium/lib9c/pull/227 PR contains [this line](https://github.com/hskim881028/lib9c/blob/4ab035a7f544804ae08c4a0729d0b3c82546d8b3/Lib9c/GameConfig.cs#L20)

Reply:
I filtered with `World.Id`, but is it better to filter with `Stage.Id`?